mixins/test/localize-mixin.test.js

```diff
@@ -126,6 +125,23 @@ const Test3LocalizeDynamicMixn = superclass => class extends LocalizeDynamicMixi
 	}
 };
 
+const Test1LocalizeHTML = superclass => class extends LocalizeStaticMixin(superclass) {
+	static get resources() {
+		return {
+			en: {
+				test1: 'This is [b]important[/b], this is [b][i]very important[/i][/b], and this is [span]not[/span]',
```

Comment:
Technically this is valid from an HTML perspective... `I <strong>like <em>jelly</strong> donuts</em>.`. Obviously this sort of thing only make sense for certain types of elements. 

Would `I [b]like [i]jelly[/b] donuts[/i]` work or would it explode? I'm not sure if I care that much about this edge case, but if it _does_ work, maybe we should have a test for it?

Reply:
The simple markup opening and closing tags are replaced separately, ignorant of their meaning or semantics, so it's just going to map to however you format it.

Reply:
Interesting - I think that's actually an error in HTML5, but will still be parsed:
https://stackoverflow.com/a/28840806

(that SO answer quotes step 8, in the spec it looks like that's changed to step 6, but still the same idea)

Reply:
Haha, I feel like I just dated myself. 😬